README.md

```diff
@@ -129,28 +129,28 @@ auto message = fine::decode<std::string>(env, term);
 
 Fine provides implementations for the following types:
 
-| Type                                 | Encoder | Decoder |
-| ------------------------------------ | ------- | ------- |
-| `fine::Term`                         | x       | x       |
-| `int64_t`                            | x       | x       |
-| `uint64_t`                           | x       | x       |
-| `double`                             | x       | x       |
-| `bool`                               | x       | x       |
-| `ErlNifPid`                          | x       | x       |
-| `ErlNifBinary`                       | x       | x       |
-| `std::string_view`                   | x       | x       |
-| `std::string`                        | x       | x       |
-| `fine::Atom`                         | x       | x       |
-| `std::nullopt_t`                     | x       |         |
-| `std::optional<T>`                   | x       | x       |
-| `std::variant<Args...>`              | x       | x       |
-| `std::tuple<Args...>`                | x       | x       |
-| `std::vector<T>`                     | x       | x       |
-| `std::map<K, V>`                     | x       | x       |
-| `fine::ResourcePtr<T>`               | x       | x       |
-| `T` with [struct metadata](#structs) | x       | x       |
-| `fine::Ok<Args...>`                  | x       |         |
-| `fine::Error<Args...>`               | x       |         |
+| Type                                                     | Encoder | Decoder |
+| -------------------------------------------------------- | ------- | ------- |
+| `fine::Term`                                             | x       | x       |
+| `int64_t`                                                | x       | x       |
+| `uint64_t`                                               | x       | x       |
+| `double`                                                 | x       | x       |
+| `bool`                                                   | x       | x       |
+| `ErlNifPid`                                              | x       | x       |
+| `ErlNifBinary`                                           | x       | x       |
+| `std::string_view`                                       | x       | x       |
+| `std::basic_string<char, std::char_traits<char>, Alloc>` | x       | x       |
+| `fine::Atom`                                             | x       | x       |
+| `std::nullopt_t`                                         | x       |         |
+| `std::optional<T>`                                       | x       | x       |
+| `std::variant<Args...>`                                  | x       | x       |
+| `std::tuple<Args...>`                                    | x       | x       |
+| `std::vector<T, Alloc>`                                  | x       | x       |
+| `std::map<K, V, Compare, Alloc>`                         | x       | x       |
```

Comment:
Should we revert this back to `std::string`, `std::vector<T>`, and `std::map<K, V>`?
While this is more accurate, it does add more cognitive load for the user.

Reply:
Yeah, lets remove extra user-side complexity.